package/version.txt

```diff
@@ -1 +1 @@
-1.14.1
\ No newline at end of file
+1.14.9.4
```

Comment:
Why this is the new version?

Reply:
since during development i had to copy from 'dist' folder to 'Pypi Server Repository' a lot of versions...